frame/democracy/src/vote.rs

```diff
@@ -131,13 +139,17 @@ impl<BlockNumber: Ord + Copy + Zero, Balance: Ord + Copy + Zero> PriorLock<Block
 }
 
 /// An indicator for what an account is doing; it can either be delegating or voting.
-#[derive(Encode, Decode, Clone, Eq, PartialEq, RuntimeDebug, TypeInfo)]
-pub enum Voting<Balance, AccountId, BlockNumber> {
+#[derive(
+	Clone, Encode, Decode, Eq, MaxEncodedLen, PartialEq, RuntimeDebug, TypeInfo,
+)]
+#[codec(mel_bound(Balance: MaxEncodedLen, AccountId: MaxEncodedLen, BlockNumber: MaxEncodedLen))]
```

Comment:
We really need a better way of handling the all-too-frequent circumstance of having a stored type one of whose params is a length bounding type for `BoundedVec`.

Reply:
```suggestion
#[codec(mel_bound(skip_type_params(MaxVotes)))]
```